src/views/portals/utils/DashboardOutingsList.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="box">
+    <h4 class="title is-3">
+      <router-link
+        :to="{
+          name: 'outings',
+          query: {
+            qa: 'draft,great',
+            bbox: '-431698,3115462,1931123,8442818',
+            offset: 0,
+            limit: 100,
+          },
+        }"
+      >
+        <icon-outing />
+        {{ $gettext('outings') | uppercaseFirstLetter }}
+      </router-link>
+    </h4>
+    <loading-notification :promise="outingsPromise" />
+    <div v-if="outingsByDate != null">
```

Comment:
Please do not use `!= null`, in most cass the value might be `undefined`, not `null`.